### PR TITLE
fix: SMTP shouldn't be configurable without IMAP enabled

### LINKED ISSUE #4311: bug: SMTP shouldn't be configurable with out IMAP enabled
Update: Yay thank you it works when I add my IMAP configuration @sojan-official! Perhaps maybe the documentation could be a bit more specific that in order to send from a certain email address they need to setup IMAP and SMTP, since maybe some people won't configure IMAP since they will just set the email forwarding rule, and when they set up SMTP they will wonder why it sends as mailer@chatwoot.com

_Originally posted by @23oijoifsd in https://github.com/chatwoot/chatwoot/issues/4263#issuecomment-1079661596_

### PR DESCRIPTION
**Pull request by aswindevps**
## Description

This change allows the user to configure SMTP only if the user configures IMAP.

Fixes #4311

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

https://user-images.githubusercontent.com/5848740/162424375-e845d3c7-3914-4714-b24d-6273e3cbc909.mp4

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/javascript/dashboard/components/SettingsSection.vue [hunk 1]
```diff
@@ -7,6 +7,7 @@
       <p class="sub-head">
         {{ subTitle }}
       </p>
+      <p v-if="note"><span class="note">Note: </span>{{ note }}</p>
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/SettingsSection.vue:10]: i18n

Comment (aswindevps) [reply app/javascript/dashboard/components/SettingsSection.vue]: Done

File: app/javascript/dashboard/components/SettingsSection.vue [hunk 2]
```diff
@@ -46,5 +54,9 @@ export default {
   .title--section {
     padding-right: var(--space-large);
   }
+
+  .note {
+    font-weight: $font-weight-bold;
```
Comment (iamsivin) [inline app/javascript/dashboard/components/SettingsSection.vue:59]:
```suggestion
    font-weight: var(--font-weight-bold);
```

Comment (aswindevps) [reply app/javascript/dashboard/components/SettingsSection.vue]: Done

**Review by iamsivin — APPROVED** [review]
LGTM⚡️

**Pull request merged**